### DISCUSSION TITLE
Dell U2720Q does not allow brightness control when DCI-PC color space setting is enabled

### DISCUSSION THREAD

**Discussion by Marcopohlo** [MonitorControl/MonitorControl#659] [General]
<img width="803" alt="Screenshot 2021-10-02 at 13 27 34" src="https://user-images.githubusercontent.com/6273948/135714135-0b642af4-bfc6-43b9-93d4-da71c844dfa6.png">

### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Brightness doesn't work at all on my U2720Q.

I've experimented with betas and also once tried to build it directly from Xcode, could this be related? I mean could some setting be altered by this experimenting?

### Expected behavior

I should be able to adjust monitor brightness through MonitorControl

### Environment Information (please complete the following information)

```markdown
- macOS version: 11.6 Big Sur
- Mac model: M1 Mini
- MonitorControl version: v3.1.1 (build 3096)
- Monitor(s): Dell U2720Q
- Monitor Cable(s)/Connection(s): USB-C
- Apple Silicon/M1 (yes or no): yes
```

**Edit:** So I tried to meddle with the settings a bit and discovered that contrast was working correctly. Then I tried to reset the monitor to factory settings and suddenly everything was working until... I changes color preset to DCI-P3.

When it's in DCI-PC, brightness doesn't work but as soon as I switch it to any other settings, everything works just fine. So sticking with sRGB for now.

Don't know how to proceed with this issue? Is this a monitor specific issue that DDC does not work when DCI-P3 is set? Or MonitorControl issue?

Thank you for the good work guys!

**Comment by waydabber** [comment]
Hi, this is definitely a monitor specific issue. DDC control seems to work but the display does not allow the setting to take place in specific color space settings.

I'll convert this to a discussion as owners of this display model might find the info important - thanks for sharing the solution!
